src/components/admin/InputGroup.tsx

```diff
@@ -9,6 +9,7 @@ export interface InputGroupProps {
   withLabel?: boolean;
   isRequired?: boolean;
   isOptional?: boolean;
+  ml?: string;
```

Comment:
Same discussion as the above `ml` prop

Reply:
[b38d47c](https://github.com/sparkletown/sparkle/pull/2930/commits/b38d47c4384bcd7ef41c720aa72ca334965f5741)